### ISSUE TITLE
Provide a documentation page

### ISSUE THREAD

**Issue by ngueno** [tamojuno/integration-api-java-sdk#23]

**Comment by efranceschi** [comment]
Concordo! A documentação da API REST é razoável, mas a documentação deste projeto java tem muito a melhorar.

Se não for possível prover uma documentação melhor à este projeto, que ao menos tenhamos alguns exemplos de como usá-la.
